split-webapp/split/src/components/TransactionList.js

```diff
@@ -15,38 +21,67 @@ function ListItemLink(props) {
   return <ListItem button component="a" {...props} />;
 }
 
+function mapDataToElements(bills) {
```

Comment:
Little bit of a nit here: Can we change this function name to something like `populateBills()` or something like that? `mapDataToElements()` is a big ambiguous to what data and elements are.